src/olympia/amo/management/commands/data_seed.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+
+from ..base import BaseDataCommand
+
+
+class Command(BaseDataCommand):
+    help = (
+        'Reset and seed the database with initial data, '
+        'generated add-ons, and data from AMO production.'
+    )
+
+    def handle(self, *args, **options):
+        num_addons = 10
+        num_themes = 5
+
+        self.clean_dir(self.data_backup_init)
+
+        self.logger.info('Resetting database...')
+        self.call_command('flush', '--noinput')
+        self.call_command('migrate', '--noinput')
+
+        self.logger.info('Loading initial data...')
+        self.call_command('loaddata', 'initial.json')
+        self.call_command('import_prod_versions')
+        self.call_command(
+            'createsuperuser',
+            '--no-input',
+            '--username',
+            settings.LOCAL_ADMIN_USERNAME,
+            '--email',
+            settings.LOCAL_ADMIN_EMAIL,
+        )
+        self.call_command('loaddata', 'zadmin/users')
+
+        self.logger.info('Generating add-ons...')
```

Comment:
You want the `reindex` with `--wipe` here as well.

Otherwise the add-ons being generated below are indexed using the automatic mapping, and without our aliases set up, which is wrong and would require a `--wipe` later to fix anyway.

Reply:
Do we need it before and after? If we call `data_load` after running the seed commands that will end with a reindex. lmk if you think we still need to wipe before.

Reply:
As it stands, until the reindex that happens in `data_load`, the indexes will be incorrect. So technically we don't need both calls if we can guarantee that everything happens uninterrupted. But I'm not sure we have that guarantee...

I think it's safer to do it here as well. But then again I'm not sure I understand why you're calling dump and load in succession in the seed command...

Reply:
Fixed. For now it's in both. I think this is generally fine and we can iterate more if we want to.